tests/functional/spec/features/view_submitted_response/view_submitted_response.js

```diff
@@ -15,6 +15,7 @@ describe("View Submitted Response", () => {
     $(AddressBlockPage.submit()).click();
     $(SubmitPage.submit()).click();
     browser.url("/submitted/view-response");
+    expect($(ViewSubmittedResponsePage.printButton()).getText()).to.equal("Print answers");
```

Comment:
I think you should assert that it is rendered on the page via like `isDisplayed()`